test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -3461,8 +3461,25 @@ Entrypoint main = bundle.js
     + 3 hidden modules"
 `;
 
+exports[`StatsTestCases should print correct stats for url 1`] = `
+"Hash: 887f827c4dce0dacbcef
+Time: Xms
+Built at: Thu Jan 01 1970 00:00:00 GMT
+                   Asset       Size  Chunks             Chunk Names
+817121871a23039712ce.jpg   10.9 KiB   {179}  [emitted]  main
+83a9c5c6df0321e72e7e.svg  656 bytes   {179}  [emitted]  main
+               bundle.js   4.51 KiB   {179}  [emitted]  main
+d01becd2595e02413b53.png     26 KiB   {179}  [emitted]  main
+Entrypoint main = bundle.js d01becd2595e02413b53.png 83a9c5c6df0321e72e7e.svg 817121871a23039712ce.jpg
+ [10] ./index.js 150 bytes {179} [built]
+[290] (webpack)/test/fixtures/images/file.png 26 KiB (url) 68 bytes (javascript) {179} [built]
```

Comment:
It might make sense to renamed `url` to `asset`

So 

`type: "asset/experimental"`

`TYPES = new Set(["asset"])`

`AssetGenerator` and `AssetJavascriptGenerator`

etc.

Opinions on that?

Reply:
Yes, I like `asset/experimental` :)